packages/common/src/store/purchase-content/sagas.ts

```diff
@@ -614,6 +614,80 @@ function* purchaseUSDCWithStripe({ amount }: PurchaseUSDCWithStripeArgs) {
   yield* put(usdcBalanceSufficient())
 }
 
+/**
+ * Collects and encrypts user's email after a successful purchase
+ * @param metadata The metadata of the purchased content
+ */
+function* collectEmailAfterPurchase({
```

Comment:
Can this live in the `sdk.tracks.purchaseTrack` and `sdk.albums.purchaseAlbum` methods?

We'd have to do something special for Coinflow's flow anyway I guess... and we don't have email there too nvm

But I do like the idea of clients being able to call `shareEmail` and pass in an email and user ID and not have to worry about the internals as much